### ISSUE TITLE
Ingestion parallel like Profiler does

### ISSUE THREAD

**Issue by duongphannamhung** [open-metadata/OpenMetadata#12222]
**Is your feature request related to a problem? Please describe.**
I've read the ingestion workflow and profiler workflow code then ingestion doesn't have a config to run in parallel, like `threadCount` in profiler give. 
Does this feature in your backlog? Or any idea pops up, hope I can give help with this.

**Comment by harshach** [comment]
@pmbrull lets make a decision on this one

**Comment by harshach** [comment]
@IceS2 add details on where we are with shipping this in 1.4.0 release

**Comment by pmbrull** [comment]
closing in favor of https://github.com/open-metadata/OpenMetadata/issues/15215
